vmdb/app/models/ext_management_system.rb

```diff
@@ -225,7 +225,7 @@ def clustered_hosts
   end
 
   def miq_proxies
-    MiqProxy.find(:all).collect {|p| p if p.ext_management_system == self}.compact
+    MiqProxy.all.select { |p| p.ext_management_system == self }
```

Comment:
This is screaming for an extra `.includes(:ext_management_system)`  I know it's shiny, but let's change in the follow up PR.